packages/@uppy/drop-target/src/index.js

```diff
@@ -1,13 +1,19 @@
-const BasePlugin = require('@uppy/core/lib/BasePlugin')
-const getDroppedFiles = require('@uppy/utils/lib/getDroppedFiles')
-const toArray = require('@uppy/utils/lib/toArray')
+import BasePlugin from '@uppy/core/lib/BasePlugin'
+import getDroppedFiles from '@uppy/utils/lib/getDroppedFiles'
+import toArray from '@uppy/utils/lib/toArray'
+
+import packageJson from '../package.json'
+
+function isFileTransfer (event) {
+  return event.dataTransfer.types?.some((type) => type === 'Files') ?? false
+}
 
 /**
  * Drop Target plugin
  *
  */
-module.exports = class DropTarget extends BasePlugin {
-  static VERSION = require('../package.json').version
+export default class DropTarget extends BasePlugin {
+  static VERSION = packageJson.version
```

Comment:
I think this might break the [babel plugin](https://www.npmjs.com/package/babel-plugin-inline-json), might be good to check

Reply:
I've recreated that plugin in the `build-lib` script: https://github.com/transloadit/uppy/blob/2a4e2bf823d37c110eb998c703d66955f131556d/bin/build-lib.js#L111-L124